docs/cpu_templates/schema.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Just out of curiosity, where does `draft-07` come from?

Reply:
ok, you're right the latest is 2020-12: https://json-schema.org/specification-links.html#2020-12 . Fixed.